src/Components/Screens/Gameplay/Chat/chat.scss

```diff
@@ -29,25 +29,25 @@
     position: relative;
     .chat-options {
       position: absolute;
-      bottom: -100px;
-      left: 10px;
+      bottom: -27px;
       display: flex;
-      flex-direction: column;
-      border-radius: 3px;
-      background-color: white;
+      border-radius: 2px;
+      padding: 5px 3px;
+      background-image: url(../../../../assets/Backgrounds/main_pj_info.png);
       box-shadow: 0px 0px 12px 3px rgb(5 5 5);
       .chat-opt {
-        color: black;
-        border-bottom: 1px solid rgba(0, 0, 0, 0.05);
+        color: white;
         padding: 0px 20px;
         font-family: 'Alegra-AO';
-        
+        background-image: url(../../../../assets/Buttons/button_red_default.png);
+        border-radius: 2px;
+        margin-inline: 3px;
```

Comment:
can you implement this without the margin-inline? its not supported in the game browser

Reply:
-replaced margin-inline: 3px, using margin: 0 2px instead.